### PR TITLE
Add initial gosheets node module

### PR DESCRIPTION
**Pull request by AlexOverbeck**
## Purpose
This creates an mostly empty node module. It includes a `gosheets.scss`
file that we can begin to use to define global styles for the "goDesign" 
design system.

_**Initializes an empty gosheets node module**_
This adds the base package lock to create a node module. All we will
need to do for this module is move sass/css files around so we will
likely not need some of the things associated with a typical module.

_**Updates node module config and adds core scss file**_
This core/gosheets.scss file will be used to import all other scss
files contained in this module.

_**Moves base scss file up a module**_
This removes the core/ directory and moves the gosheets.scss file up
to the root directory. The gosheets.scss file now also includes a @warn
to ensure that sass is actually working when the file is imported.

### CODE REVIEW

**Review by grahamhency — APPROVED** [review]
This is a nice change I'm glad we're moving towards. These all look good to me. :+1:

File: gosheets.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+@warn 'gosheets!!!!';
```
Comment (devchris) [inline gosheets.scss]: Should we still add an empty line at the end of the file?

**Review by devchris — APPROVED** [review]
Some nice changes are going to come in the near future 💥

File: gosheets.scss (hunk 1 above)
Comment (AlexOverbeck) [reply gosheets.scss]: Yep, I will rebase that in that change before I merge.

**Review by StevenUlmer — APPROVED** [review]
Love where we're going with this!

**Pull request merged**